nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc

```diff
@@ -15,7 +15,21 @@ Alternatively, to share credentials between objects in multiple namespaces, you
 
 Trigger authentications and cluster trigger authentication use the same configuration. However, a cluster trigger authentication requires an additional `kind` parameter in the authentication reference of the scaled object.
 
-.Example trigger authentication with a secret
+.Example secret for Basic Authentication
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keda-prom-secret
+  namespace: default
+data:
+  username: "dXNlcm5hbWUK" <1>
+  password: "cGFzc3dvcmQK"
```

Comment:
It looks like your example username and password include a binary newline character. Not that it's super important, but could be confusing. Using `echo -n` will cause echo not to send a trailing newline character and the resulting base64 strings have `=` instead of `K` for their final character.
```
$ echo -n username | base64
dXNlcm5hbWU=
$ echo -n password | base64
cGFzc3dvcmQ=
```